stl/inc/numbers

```diff
@@ -87,86 +87,42 @@ namespace numbers {
     template <floating_point _Floating>
     inline constexpr _Floating phi_v<_Floating> = static_cast<_Floating>(1.618033988749895);
 #else // ^^^ defined(__cpp_lib_concepts) / !defined(__cpp_lib_concepts) vvv
-    template <>
-    inline constexpr double e_v<double> = 2.718281828459045;
-    template <>
-    inline constexpr double log2e_v<double> = 1.4426950408889634;
-    template <>
-    inline constexpr double log10e_v<double> = 0.4342944819032518;
-    template <>
-    inline constexpr double pi_v<double> = 3.141592653589793;
-    template <>
-    inline constexpr double inv_pi_v<double> = 0.3183098861837907;
-    template <>
-    inline constexpr double inv_sqrtpi_v<double> = 0.5641895835477563;
-    template <>
-    inline constexpr double ln2_v<double> = 0.6931471805599453;
-    template <>
-    inline constexpr double ln10_v<double> = 2.302585092994046;
-    template <>
-    inline constexpr double sqrt2_v<double> = 1.4142135623730951;
-    template <>
-    inline constexpr double sqrt3_v<double> = 1.7320508075688772;
-    template <>
-    inline constexpr double inv_sqrt3_v<double> = 0.5773502691896257;
-    template <>
-    inline constexpr double egamma_v<double> = 0.5772156649015329;
-    template <>
-    inline constexpr double phi_v<double> = 1.618033988749895;
+    template <class _Ty>
+    struct _Reject_primary_template {

```

Comment:
Minor request - could this instead be `_Reject_invalid` or something along those lines? I think `_Reject_primary_template` is kind of a confusing name.